### PR TITLE
WNB-62: Backend file structure

### PR DESCRIPTION
**Pull request by Fralou**
https://wannabook.atlassian.net/browse/WNB-62

Doc with file structure: https://docs.google.com/document/d/105KeadpiZJVSxqKhM0tvL6dNR5x1DZviW6qQvqorJt4/edit#heading=h.ipnzu6ut8ydi

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "wannabook-api",
+  "version": "1.0.0",
+  "description": "Wannabook project",
+  "main": "app.js",
+  "dependencies": {
+    "body-parser": "^1.18.3",
```
Comment (ikushlianski) [inline package.json]: Let's use strict versions.

I did `npm config set save-exact true` on my machine so my npm always installs strict versions

File: routes/test.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', (req, res) => {res.json({msg: "Wannabook /test route"})});
+
+module.exports = router;
```
Comment (ikushlianski) [inline routes/test.js]: I would put empty line at the end of every file

File: app.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+const app = express();
+
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+app.get('/', (req, res) => res.json({msg: 'Wannabook!'}));
+
+app.use('/test', test);
+const port = process.env.PORT || 5000;
```
Comment (ikushlianski) [inline app.js]: We should later outsource port names and other stuff into configs.
See https://wannabook.atlassian.net/browse/WNB-70

**Review by ikushlianski — CHANGES_REQUESTED** [review]

File: app.js (hunk 3 above)
Comment (letutsenik) [reply app.js]: Please, use [config](https://www.npmjs.com/package/config) package to handle different enviroments.

Comment (ikushlianski) [reply app.js]: Yes, agree. Could you create some very basic config in this repo for now?

**Review by ikushlianski — APPROVED** [review]

**Review by letutsenik — APPROVED** [review]

**Pull request merged**